carbonmark/components/pages/Project/Retire/TotalValues.tsx

```diff
@@ -0,0 +1,285 @@
+import { cx } from "@emotion/css";
+import { trimWithLocale } from "@klimadao/lib/utils";
+import { t } from "@lingui/macro";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import { Text } from "components/Text";
+import { getConsumptionCost, getFeeFactor } from "lib/actions.retire";
+import { AGGREGATOR_FEE, CARBONMARK_FEE, SUSHI_SWAP_FEE } from "lib/constants";
+import { formatToPrice, formatToTonnes } from "lib/formatNumbers";
+import { carbonmarkPaymentMethodMap } from "lib/getPaymentMethods";
+import { Price } from "lib/types/carbonmark";
+import Image from "next/legacy/image";
+import { useRouter } from "next/router";
+import { FC, useEffect, useState } from "react";
+import { useFormContext, useWatch } from "react-hook-form";
+import * as styles from "./styles";
+import { FormValues } from "./types";
+
+type TotalValuesProps = {
+  singleUnitPrice: Price["singleUnitPrice"];
+  balance: string | null;
+  pool: Lowercase<Price["name"]>;
+};
+
+export const TotalValues: FC<TotalValuesProps> = (props) => {
+  const { locale } = useRouter();
+  const { formState, control, setValue } = useFormContext<FormValues>();
+  const [isLoading, setIsLoading] = useState(false);
+  const [costs, setCosts] = useState("");
+  const [feesFactor, setFeesFactor] = useState(0);
+  const [error, setError] = useState("");
+  const [isToggled, setIsToggled] = useState(false);
+
+  const amount = useWatch({ name: "quantity", control });
+  const paymentMethod = useWatch({ name: "paymentMethod", control });
+
+  const redemptionFee = Number(costs || 0) * feesFactor;
+  const aggregatorFee = Number(amount || 0) * AGGREGATOR_FEE;
+  const swapFee = Number(costs || 0) * SUSHI_SWAP_FEE;
```

Comment:
This changes based on the destination pool -- 2 swaps to get to BCT

Reply:
So just the swap fee * 2 if pool is "bct"?
Done.